src/client/pages/home/Home.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import FlexBox from "components/flexbox/FlexBox";
+import Tetromino from "components/tetrominoes/Tetrominoes";
+import { StoreContext } from "store";
+import { grow } from "actions/store";
+
+function timeout(ms) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+export default function Home() {
+  const { state, dispatch } = React.useContext(StoreContext);
+
+  const incrementAge = async () => {
+    await timeout(1000);
+    dispatch(grow(1));
+  };
+
+  console.log(state);
+  return (
+    <FlexBox height="full" className="items-center justify-center">
```

Comment:
Once again, just to know, how easy is it with the FlexBox Tetrominoes to place them?
Like to place them side to side, or to "emboite" them.

It looks good tho.

Reply:
I'm not sure that it's usable in real condition (i.e. in game) without relying on complex logic with DOM.
I think we should work with w-x/y and h-x/y from [Tailwind](https://tailwindcss.com/docs/width#app).